### PR TITLE
Dev Team: Setting up Rspec

### PR DESCRIPTION
**Pull request by ESteinman**
### User story
```
As a development team,
In order to make unit testings
I would like to use Rspec as my Unit testing framework.
```
### What've learned
Change Ruby-version

### CODE REVIEW

**Comment by faraznaeem** [comment]
- Missing PT link

File: spec/rails_helper.rb [hunk 1]
```diff
@@ -0,0 +1,22 @@
...
+RSpec.configure do |config|
+  config.fixture_path = "#{::Rails.root}/spec/fixtures"
+
+  config.use_transactional_fixtures = true
+  config.infer_spec_type_from_file_location!
+
+  config.filter_rails_from_backtrace!
+end
+
+Shoulda:Magchers.configure do |config|
```
Comment (faraznaeem) [inline spec/rails_helper.rb]: Typo

File: spec/spec_helper.rb [hunk 2]
```diff
@@ -0,0 +1,96 @@
...
+  #     --seed 1234
+  config.order = :random
+
+  # Seed global randomization in this process using the `--seed` CLI option.
+  # Setting this allows you to use `--seed` to deterministically reproduce
+  # test failures related to randomization by passing the same `--seed` value
+  # as the one that triggered the failure.
+  Kernel.srand config.seed
+=end
+end
```
Comment (faraznaeem) [inline spec/spec_helper.rb]: clean comments

**Review by faraznaeem — CHANGES_REQUESTED** [review]

File: spec/rails_helper.rb (hunk 1 above)
Comment (ESteinman) [reply spec/rails_helper.rb]: Done

File: spec/spec_helper.rb (hunk 2 above)
Comment (ESteinman) [reply spec/spec_helper.rb]: Done

**Review by faraznaeem — APPROVED** [review]

**Pull request merged**